setup/ci/gcb.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: single
+title:  "Google Cloud Build"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Setting up [Google Cloud Build](https://cloud.google.com/cloud-build/) as a Continuous Integration (CI)
+system within Spinnaker enables triggering pipelines when builds complete.
+
+## Prerequisites
+
+You need to have a [Google Cloud Platform](https://cloud.google.com) project with the
+[Cloud Build API](http://console.cloud.google.com/apis/library/cloudbuild.googleapis.com) enabled.
+You can enable the API with the following `gcloud` command:
+
+```
+gcloud services enable cloudbuild.googleapis.com
+```
+
+## Configure Spinnaker to Listen for Google Cloud Build Pub/Sub Notifications
+
+Google Cloud Build sends [Build Notifications](https://cloud.google.com/cloud-build/docs/send-build-notifications)
+when the state of your build changes. You can create a pipeline trigger to invoke a pipeline based
+on the status of your build.
+
+1. Create a Subscription object for the `cloud-builds` topic in your project:
+
+    ```
+    PROJECT_ID=
+    SUBSCRIPTION_NAME=googleCloudBuilds
+
+    gcloud pubsub subscriptions create $SUBSCRIPTION_NAME --topic projects/$PROJECT_ID/topics/cloud-builds
+    ```
+
+2. Configure and deploy Halyard with your newly created subscription.
+
+    ```
+    hal config pubsub google subscription add $SUBSCRIPTION_NAME --project $PROJECT_ID --subscription-name $SUBSCRIPTION_NAME
+
+    hal config pubsub google enable
+
+    hal deploy apply
+    ```
+
+## Configure Your Pipeline Trigger
```

Comment:
Any way we could extract artifact information from the trigger?

/cc @ezimanyi

Reply:
Yes with the right jinja template. If argo changes their payload to include "digest" (important for reproducible deployments) we will need to change this template.

Reply:
I'm working with Clod Build team to establish payload. We'll package a default jinja template with echo thereafter.

Reply:
Here's the schema of the message payload - what fields do we need from it to make artifacts work?

https://cloud.google.com/cloud-build/docs/api/reference/rest/v1/projects.builds#Build

Reply:
@lwander and I spoke with David, and he pointed out where the digest is contained in the pubsub message, so Lars is going to proceed with adding the jinja template.